src/ScheduleService.Business/Group/Interfaces/IUpdateGroupCommand.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Group;
+using Microsoft.AspNetCore.JsonPatch;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ScheduleService.Business.Group.Interfaces;
+
+[AutoInject]
+public interface IUpdateGroupCommand
```

Comment:
make edit for patch command and put for update(optional)

Reply:
+